src/main/java/io/github/thebusybiscuit/slimefun4/core/networks/cargo/CargoNet.java

```diff
@@ -294,7 +294,8 @@ private static int getFrequency(Location l) {
             return Integer.parseInt(str);
         }
         catch (Exception x) {
-            Slimefun.getLogger().log(Level.SEVERE, "An Error occured while parsing a Cargo Node Frequency", x);
+            Slimefun.getLogger().log(Level.SEVERE, "An Error occured while parsing Cargo Node Frequency "
```

Comment:
Should probably use a deferred error message by using a lambda here.
Otherwise Sonarcloud will kill ya